docs/semgrep-code/overview.md

```diff
@@ -0,0 +1,44 @@
+---
+slug: overview
+append_help_link: true
+title: Overview
+hide_title: false
+description: Learn about Semgrep Code, a static application security testing (SAST) tool |
+    designed to find security vulnerabilities in your first-party code.
```

Comment:
```suggestion
    that finds security vulnerabilities in your first-party code.
```

Extremely minor nit, you can keep the original wording if you like, this is just a cut-down-words suggestion.